src/app-layout/visual-refresh/app-bar.tsx

```diff
@@ -2,8 +2,9 @@
 // SPDX-License-Identifier: Apache-2.0
 import React from 'react';
 import clsx from 'clsx';
-import { useAppLayoutInternals } from './context';
 import { InternalButton } from '../../button/internal';
+import { MobileTriggers as DrawersMobileTriggers } from './drawers';
```

Comment:
I do not think we should isolate the feature in a separate file. It becomes a first class member (even if not exposed via API yet), so the new code should seamlessly integrate in the current implementation

Reply:
Per previous comment this will be addressed in a follow up PR that resolves many of the above comments.